ApnsPHP/Push.php

```diff
@@ -638,6 +638,12 @@ private function httpSend(Message $message, &$reply): bool
             CURLOPT_POSTFIELDS => $message->getPayload()
             ]) && ($reply = curl_exec($this->hSocket)) !== false)
         ) {
+            $errorCode = curl_errno($this->hSocket);
+
+            if ($errorCode > 0) {
+                $reply = sprintf('Response: %s; Error code: %d; Error message: %s.', $reply, $errorCode, curl_error($this->hSocket));
```

Comment:
Shouldn't this use the logger? I don't see how `$reply` can be used any further.

Reply:
We pass the `$reply` parameter by reference (&) and then use it in error log 
https://github.com/move-backend/ApnsPHP/blob/master/ApnsPHP/Push.php#L577

Reply:
Ah you're right. `$reply` seems to have a very specific meaning though, and I don't think we should use it for general logging. Let's just use the logger here.

Reply:
We just pass it to `statusMessage` log field but okay replaced the reply usage with the logger call.